misc/container-metadata-file-validator-windows/container-metadata-file-validator-windows.dockerfile

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+FROM microsoft/windowsservercore:latest
```

Comment:
Can we use microsoft/nanoserver for this?

Reply:
Yes! Decreases the image size from 11gb to 1.1gb!